package.json

```diff
@@ -12,13 +12,19 @@
     "build:vm": "yarn workspace near-mock-vm run build:wasm",
     "test": "yarn workspaces run test && yarn check:pretty",
     "doc": "rm -rf docs; typedoc --tsconfig tsconfig.json && touch docs/.nojekyll",
-    "prettify": "prettier --write '**/*.ts'",
-    "check:pretty": "yarn prettier --check '**/*.ts'"
+    "prettify": "prettier --write '**/src/**/*.ts'",
```

Comment:
You mentioned that prettier mostly doesn't work; why not remove it?

Reply:
It only doesn't work for AS.  TS is still fine.   I could have combined the two into ESLint this is what the AS compiler does't, but I don't need any fancy rules for TS like the compiler does (it needs to be abled to be compiled with AS).  So this keeps it simple on the TS side.